### ISSUE TITLE
Org Managers

### ISSUE THREAD

**Issue by n8tlarsen** [ra-rs/ra#6]
I'm not able to manage this repository at this time. Please reply to this issue if you would like to be added to the ra-rs org and assume a managerial role in this project.

**Comment by elrafoon** [comment]
Hello, I'm quite busy too, but my company uses RA6M3 on many projects and we are spearheading it's programming in Rust lang. Right now I'm porting rustBoot to RA6M3, that's how I came to this project.

I would hate to see this project rot, so if no-one with more free time steps in, I'm willing to manage it.

**Comment by elfmimi** [comment]
@n8tlarsen ping!

**Comment by AddisonH** [comment]
@n8tlarsen I'm an application engineer at Renesas. I'd be interested.

**Comment by elrafoon** [comment]
That would be great. We are already using Rust for Renesas/RA coding, but we really miss the PAC and embassy support that many other platforms enjoy. We use ThreadX/NetX/FSP and custom wrappers, but it's not anywere close to the experience I got while coding Nordic NRF chips using pure Embassy.

We'd be happy to help, we have plenty of experience with Rust on the RA.

@AddisonH are you going to maintain the crate?

@n8tlarsen wake up or we fork!  :-)

**Comment by AddisonH** [comment]
@elrafoon Yes! @n8tlarsen has added me to the org. I've been on vacation but I will be maintaining this crate now.

**Comment by n8tlarsen** [comment]
@elrafoon I also added you to the org. Please check to make sure you both (@AddisonH) can push to the repo.

**Comment by elrafoon** [comment]
Thanks @n8tlarsen !

**Comment by n8tlarsen** [comment]
@elrafoon it looks like you have not yet accepted the membership invite, please do so to get write access to the ra-rs repo.

**Comment by elrafoon** [comment]
Sorry @n8tlarsen I overlooked the invitation, so I didn't accept it. It was valid till yesterday, so please can you send me it once again? Thanks.

**Comment by felipebalbi** [comment]
Hi folks,

Sorry to hijack the issue, but are there any updates on this topic? I'm trying to use the RA4M1 pac but it needs some updates. I patched the python script, but when running with recent `svdtools` it fails to render the pacs. Interested in using an RA4M1 board with Rust, and willing to help the project :)

**Comment by AddisonH** [comment]
Hey there. That is correct about the pac version argument. I'll document that somewhere. There's a problem with the latest version of SVD tools and the Renesas SVD files. I opened an issue and some of the problems have been resolved but there's still something else to track down yet. I've been working on it when I have time but I've been busy with other stuff at work lately. Hopefully I'll have some updates over the holidays!

**Comment by nguyentri** [comment]
Hi @n8tlarsen @AddisonH 

This is Tri, an application engineer at Renesas.
I have started learning Rust recently and have a plan to implement embassy-ra with your RA crates.
However, I got some troubles when working with like https://github.com/ra-rs/ra/pull/7.
I am willing to co-maintain this repo to bring up Rust to Renesas products.

If possible, can you add me to maintainer for this repo?

Best Regards,
Tri Nguyen

**Comment by n8tlarsen** [comment]
@nguyentri I sent the org member invite. Please accept to be added.

**Comment by nguyentri** [comment]
@n8tlarsen 
Thank you for your invitation.
However, I have created other PACs for [RA series](https://github.com/nguyentri/ra-pac) and [DA series](https://github.com/nguyentri/da-pac) with new tool [svd2pac](https://github.com/Infineon/svd2pac) as offered some benefits.
